platform/linux-generic/odp_stash.c

```diff
@@ -357,7 +480,7 @@ odp_stash_t odp_stash_lookup(const char *name)
 	return ODP_STASH_INVALID;
 }
 
-static inline int32_t stash_put(odp_stash_t st, const void *obj, int32_t num)
+static inline int32_t stash_put(odp_stash_t st, const void *obj, int32_t num, odp_bool_t batch)
```

Comment:
`is_batch` would be a bit more boolean-like.

(Though, current style seems to be `batch` with ones and zeros, well, up to you.)

Reply:
Fixed on V2.